platform/config/service/standard/gateway-svc.yml.in

```diff
@@ -73,27 +73,6 @@ spec:
                         port: 8889
```

Comment:
The `/` liveness probe no longer functions (404s). You need to add a /v1/ping endpoint and change the liveness probe.

Reply:
This is fixed